scenariolib/event_search.go

```diff
@@ -12,20 +12,19 @@ import (
 
 // SearchEvent a struct representing a search, is defined by a query to execute
 type SearchEvent struct {
-	query string
-	// keyword exists because the query sent to the index may be different than the keyword(s) used to search
+	query       string
+	// keyword exists because the query sent to the index may be diffrent than the keyword(s) used to search
 	keyword     string
 	actionCause string
 	actionType  string
 	logEvent    bool
 	customData  map[string]interface{}
 }
 
-func newSearchEvent(e *JSONEvent, c *Config) (*SearchEvent, error) {
+func newSearchEvent(e *JSONEvent, c *Config, v *Visit) (*SearchEvent, error) {
```

Comment:
I have a concern about that actually.  Either the language should be in the JSONEvent Arguments, or there should be a mechanism to obtain the visit from everywhere since it has some global informations like the current language of the visitor.